src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py

```diff
@@ -8,18 +8,14 @@
 from tempfile import gettempdir
 from uuid import uuid4
 
-path = '/usr/share/archivematica/dashboard'
-if path not in sys.path:
-    sys.path.append(path)
+import django
+sys.path.append('/usr/share/archivematica/dashboard')
```

Comment:
This was fixed in #161 (after I forgot this conversation and did the work anyway)